app/src/org/commcare/android/view/TabbedDetailView.java

```diff
@@ -183,5 +196,30 @@ public int getCurrentTab() {
     public int getTabCount() {
         return mViewPager.getAdapter().getCount();
     }
-    
+
+    //region Private classes
+
+    private class DefaultEDVModifier implements EntityDetailAdapter.EntityDetailViewModifier, Parcelable {

```

Comment:
Got this lint warning:
According to the Parcelable interface documentation, "Classes implementing the Parcelable interface must also have a static field called CREATOR, which is an object implementing the Parcelable.Creator interface.

Might be worth looking into